### PR TITLE
Warn user before deleting a mint with dialogue box.

### PR DESCRIPTION
**Pull request by Taseeeer**
Screenshot 2023-06-11 at 3 33 22 PM

### CODE REVIEW

File: package-lock.json
Comment (aurorabor) [inline package-lock.json:1]: I don't see why these changes are necessary

Comment (Taseeeer) [reply package-lock.json:1]: @aurorabor is this not how you warn a user with a dialogue box?

**Comment by callebtc** [comment]
Very cool!

File: src/components/SettingsView.vue [hunk 1]
```diff
@@ -196,6 +196,24 @@
...
+    v-model="showRemoveMintDialog"
+  >
+    <q-card class="q-pa-lg">
+      <h6 class="q-my-md text-primary">Are you sure you want to delete this mint?</h6>
+      <div class="row q-mt-lg">
+        <q-btn
+          outline
+          v-close-popup
+          color="primary"
+          icon="remove"
```
Comment (callebtc) [inline src/components/SettingsView.vue:209]:
```suggestion
```

**Comment by callebtc** [comment]
Just removed the icon because it looks a bit wonky on that button. LGTM
